### ISSUE TITLE
Deprecation warnings for invalid escape sequence

### ISSUE THREAD

**Issue by Hieronimo** [alberanid/imdbpy#282]
I get this warnings everytime I use _imdbpy_:
```
..../imdb/utils.py:74: DeprecationWarning: invalid escape sequence \s
  re_episodes = re.compile('\s?\((\d+) episodes\)', re.I)
..../imdb/utils.py:601: DeprecationWarning: invalid escape sequence \(
  name = re.sub('[!@#$\(\)\[\]]', '', name)
..../imdb/parser/http/companyParser.py:36: DeprecationWarning: invalid escape sequence \s
  _re_company_name = re.compile('With\s+(.+)\s+\(Sorted by.*', re.I | re.M)
..../imdb/parser/http/piculet.py:730: DeprecationWarning: invalid escape sequence \s
  'clean': lambda xs: re.sub('\s+', ' ', ''.join(xs).replace('\xa0', ' ')).strip(),
..../imdb/parser/http/movieParser.py:1452: DeprecationWarning: invalid escape sequence \s
  re_means = re.compile('mean\s*=\s*([0-9]\.[0-9])\s*median\s*=\s*([0-9])', re.I)
..../imdb/parser/http/movieParser.py:1914: DeprecationWarning: invalid escape sequence \|
  (re.compile('<span class="ghost">\|</span>', re.I), r':::'),
..../imdb/parser/http/personParser.py:443: DeprecationWarning: invalid escape sequence \s
  (re.compile('(</table>\n</div>\s+)</div>', re.I + re.DOTALL), r'\1'),
..../imdb/parser/http/personParser.py:445: DeprecationWarning: invalid escape sequence \.
  (re.compile('\.<br><br>([^\s])', re.I), r'. \1')
```

**Comment by tirkarthi** [comment]
pyupgrade automatically fixes these : https://github.com/asottile/pyupgrade/

**Comment by tirkarthi** [comment]
Full list of warnings as below. I have created #286 to fix this.

```
find . -iname '*.py' | grep -v example  | xargs -P4 -I{} python3 -Wall -m py_compile {}
./imdb/utils.py:74: DeprecationWarning: invalid escape sequence \s
  re_episodes = re.compile('\s?\((\d+) episodes\)', re.I)
./imdb/utils.py:601: DeprecationWarning: invalid escape sequence \(
  name = re.sub('[!@#$\(\)\[\]]', '', name)
./imdb/parser/http/personParser.py:440: DeprecationWarning: invalid escape sequence \s
  (re.compile('(</table>\n</div>\s+)</div>', re.I + re.DOTALL), r'\1'),
./imdb/parser/http/personParser.py:442: DeprecationWarning: invalid escape sequence \.
  (re.compile('\.<br><br>([^\s])', re.I), r'. \1')
./imdb/parser/http/movieParser.py:1433: DeprecationWarning: invalid escape sequence \s
  re_means = re.compile('mean\s*=\s*([0-9]\.[0-9])\s*median\s*=\s*([0-9])', re.I)
./imdb/parser/http/movieParser.py:1895: DeprecationWarning: invalid escape sequence \|
  (re.compile('<span class="ghost">\|</span>', re.I), r':::'),
./imdb/parser/http/companyParser.py:36: DeprecationWarning: invalid escape sequence \s
  _re_company_name = re.compile('With\s+(.+)\s+\(Sorted by.*', re.I | re.M)
./imdb/parser/http/piculet.py:730: DeprecationWarning: invalid escape sequence \s
  'clean': lambda xs: re.sub('\s+', ' ', ''.join(xs).replace('\xa0', ' ')).strip(),
./imdb/helpers.py:531: DeprecationWarning: invalid escape sequence \(
  _re_akas_country = re.compile('\(.*?\)')
./docs/goodies/download_applydiffs.py:175: DeprecationWarning: invalid escape sequence \.
  if re.match(".*\.list\.gz",f) or re.match(".*\.list",f):
./docs/goodies/download_applydiffs.py:175: DeprecationWarning: invalid escape sequence \.
  if re.match(".*\.list\.gz",f) or re.match(".*\.list",f):
./docs/goodies/download_applydiffs.py:301: DeprecationWarning: invalid escape sequence \.
  if re.match(".*\.list\.gz",f):
./docs/goodies/download_applydiffs.py:363: DeprecationWarning: invalid escape sequence \.
  if re.match(".*\.list",f):
./docs/goodies/download_applydiffs.py:440: DeprecationWarning: invalid escape sequence \.
  if re.match(".*\.list",f):
./docs/goodies/download_applydiffs.py:452: DeprecationWarning: invalid escape sequence \.
  if re.match(".*\.list.gz",f):
```

**Comment by alberanid** [comment]
Fixed by Karthikeyan Singaravelan in #286
